src/styles/_constants.scss

```diff
@@ -1,11 +1,11 @@
 $polaris-bg-default: #FFFFFF, #E1E0E4, #FFFFFF;
```

Comment:
Should be grey to white to grey. (ref: https://app.zeplin.io/project.html#pid=56cf0a7e3173092129bdfcf4&sid=56cf0adaddc4c65742e414de)
```$polaris-bg-default: #e5e5ea, #FFFFFF, #e5e5ea;```